snapcraft/tests/test_pluginhandler.py

```diff
@@ -340,9 +341,93 @@ def test_stage_state(self):
     def test_strip_state(self):
         self.assertEqual(None, self.handler.last_step())
 
+        bindir = os.path.join(self.handler.code.installdir, 'bin')
+        os.makedirs(bindir)
+        open(os.path.join(bindir, '1'), 'w').close()
+        open(os.path.join(bindir, '2'), 'w').close()
+
+        self.handler.mark_done('build')
+        self.handler.stage()
         self.handler.strip()
 
         self.assertEqual('strip', self.handler.last_step())
+        with open(self.handler._step_state_file('strip'), 'r') as f:
+            state = yaml.load(f)
+
+        self.assertTrue(state, 'Expected strip to save state YAML')
```

Comment:
Ah, this is a carry-over from before I had state classes, so this line is redundant anyway.